pymllm/tests/bench_cpu_busy_loop.py

```
@@ -0,0 +1,113 @@
+    sock.bind(addr)
+
+    poller = zmq.Poller()
+    poller.register(sock, zmq.POLLIN)
+
+    duration = 3.0  # seconds per test
+
+    print("=" * 64)
+    print("Scheduler CPU Busy-Loop Benchmark")
+    print("=" * 64)
+    print(f"Each test runs for {duration:.0f}s simulating the scheduler poll loop")
+    print(f"(poll for requests → loop back, no simulated GPU work)")
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)